docs/install.md

```diff
@@ -105,6 +105,100 @@ grep /etc/group -e "docker"
 sudo systemctl restart coder.service
 ```
 
+## Kubernetes via Helm
+
+Before proceeding, please ensure that you have both Helm 3.5+ and the
+[latest version of Coder](https://github.com/coder/coder/releases) installed.
+You will also need to have a Kubernetes cluster running K8s 1.19+.
+
+> See our [Helm README](https://github.com/coder/coder/blob/main/helm#readme)
+> file for additional information.
+
+> ⚠️ **Warning**: Helm support is new and not yet complete. There may be changes
+> to the Helm chart between releases which require manual values updates. Please
+> file an issue if you run into any issues.
+>
+> Additionally, the Helm chart does not currently automatically configure a
+> Service Account and workspace template for use in Coder. See
+> [#3265](https://github.com/coder/coder/issues/3265).
+
+
+1. Create a PostgreSQL deployment. Coder does not manage a database server for
```

Comment:
This is a huge addition to the setup complexity. Can we default to the embedded Postgres?

Reply:
"Huge" may be an overstatement. It is an extra steps and a set of decisions the user must make.

Reply:
The embedded postgres in v2 does not support HA. The reasoning for not including a postgres deployment in our helm chart like we did in v1 was because the timescale deployment in v1 was ONLY for trials and not for production deployments. We have a better way of supporting trials in v2: running the coder binary directly on the host

I believe we should treat the helm chart in v2 as a production-only thing. People *can* use it for trials, but they're gonna have to do a full deployment and won't have some of the niceties you get when running coder directly or via docker-compose

Reply:
Ok, all makes sense.

To make it easier for the user, how about we give them the specific commands they could use to apply the Bitnami postgres helm chart? I'm pushing for the most brainless experience as possible.

Reply:
How's this @ammario

Reply:
SGTM